readit/database.py

```diff
@@ -30,7 +31,15 @@ def init_db(self, cursor, db):
         """
 
         try:
-            self.db = sqlite3.connect('test.db')
+            path = os.path.expanduser('~') + "/.config/readit"
```

Comment:
You should use `os.path.join` instead of string concatenation 
https://stackoverflow.com/a/13944874
And it would be more readable if we use `config_path` variable name instead of just `path`